frontend/app/components/collapsible.tsx

```diff
@@ -0,0 +1,48 @@
+import type { ComponentProps } from 'react';
+
+import { cn } from '~/utils/tw-utils';
+
+const variants = {
```

Comment:
Remove this since it's unused.

Reply:
Isn't it used on line 30?

Reply:
yes, but variant is not a `details` and `summary` HTML tag prop. I think you copy paste that from another compoment.

Reply:
sorry I mean the `cn` part.

Reply:
ok, the comment was on the `variants` variable.